src/amber/environment/settings.cr

```diff
@@ -1,10 +1,18 @@
-require "logger"
+require "./logger"
 require "yaml"
 
 module Amber::Environment
   class Settings
-    property logger : ::Logger = Logger.new(STDOUT)
-    property colorize_logging : Bool
+    alias LoggingType = NamedTuple(severity: String,
```

Comment:
Could this be cleaned up a bit?

Reply:
We can break the subclasses into its own file. Or do you mean clean the **LoggingType**?

Reply:
I meant with logging type.

It would be more readable and accomplish the same thing to just set logger as a default named tuple.
`property logging = {severity: "info, color: true, time: false, level: false, etc}`

Then all the values would still have to match that.

Reply:
I did it with the logging type with the thought that we do not want to default at the property level and only default within the YAML

Reply:
I agree. It's just that defining a NamedTuple type looks even uglier. I'm fine with it though.